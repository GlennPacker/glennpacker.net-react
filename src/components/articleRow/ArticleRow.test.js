import React from 'react';
import ArticleRow from './index';
import { shallow } from 'enzyme';

describe('ArticleRow', () => {
    it('renders correctly', () => {
        const data = ['a', 'b'];

        const wrapper = shallow(
            <ArticleRow
                minHeight="100"
                data={data}
            />
        )
        expect(wrapper.find('.col-md').length).toBe(2)
        expect(wrapper).toMatchSnapshot();
    })
})