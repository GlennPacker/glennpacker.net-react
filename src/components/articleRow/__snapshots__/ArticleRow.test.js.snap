// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleRow renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-md"
    key="0"
  >
    <Article
      0="a"
      minHeight="100"
    />
  </div>
  <div
    className="col-md"
    key="1"
  >
    <Article
      0="b"
      minHeight="100"
    />
  </div>
</div>
`;
